api/user/comments/comments.py

```diff
@@ -8,11 +11,53 @@
 @comments_bp.route("/<string:author_id>/posts/<string:post_id>/comments", methods=["GET"])
 def get_comments(author_id: str, post_id: str):
     """get the list of comments of the post whose id is POST_ID (paginated)"""
-    pagination = get_pagination_params()
-    return {"comments": ["", ""]}
+
+    post = Post.query.filter_by(id=post_id).first()
+
+    if not post:
+        return {"message": "Post not found."}, 404
+
+    comments = post.comments.paginate(**get_pagination_params().as_dict()).items
+    comment_list = []
+
+    # If author (local or remote) is missing, skip that comment
+    for comment in comments:
+        comment_json = comment.getJSON()
+        if not comment_json["author"]:
+            continue
+        comment_list.append(comment_json)
+
+    json = {
+        "type": "comments",
+        "page": get_pagination_params().page,
+        "size": get_pagination_params().per_page,
+        "post": post.url,
+        "id": post.url + "/comments",
+        "comments": comment_list,
+    }
+
+    return json
 
 
 @comments_bp.route("/<string:author_id>/posts/<string:post_id>/comments", methods=["POST"])
 def post_comment(author_id: str, post_id: str):
     """if you post an object of “type”:”comment”, it will add your comment to the post whose id is POST_ID"""
-    pass
+
+    data = request.json
+
+    try:
+        comment = Comment(
+            id=data["id"],
+            published=data["published"],
+            comment=data["comment"],
+            contentType=data["contentType"],
+            post_url=data["id"][: data["id"].index("comments") - 1],
```

Comment:
slicing like this makes me nervous :| but the spec for IDs like this is kinda gross anyway, not much we can do